docs/Basic-Functions/Explorer-tab.md

```diff
@@ -0,0 +1,75 @@
+# Explorer tab
+
+!!! Info
+
+    By default, when the page loads it will always be on the Explorer Tab. This tab allows users to quickly navigate and open all datasets, recently viewed chromosomes, or favorite datasets. It also enables users within a specific group to access and use data categorized under that group, which is typically not public.
+
+![Explorer-tab-Static-Revised](https://github.com/user-attachments/assets/a24e873f-8240-4a4f-adf2-e6e40300a2d6)
+
+## Datasets included in Pretzel
+
+List of the _Triticum aestivum_, _Hordeum vulgare_ and _Cicer arietinium_ datasets currenty housed in Pretzel could be assessed using the following link:
+
+``` https://agg.plantinformatics.io/
```

Comment:
A link can be added like like this

`[agg.plantinformatics.io](https://agg.plantinformatics.io/)`